lightning/restore/restore.go

```diff
@@ -672,8 +672,35 @@ var gcLifeTimeKey struct{}
 func (rc *RestoreController) restoreTables(ctx context.Context) error {
 	logTask := log.L().Begin(zap.InfoLevel, "restore all tables data")
 
-	var wg sync.WaitGroup
+	// for importer/local backend, we should disable some pd scheduler and change some settings, to
+	// make split region and ingest sst more stable
+	if rc.cfg.TikvImporter.Backend != config.BackendTiDB {
+		// disable some pd schedulers
+		opt := pd.SecurityOption{}
+		if rc.cfg.Security.CAPath != "" {
+			opt.CAPath = rc.cfg.Security.CAPath
+			opt.CertPath = rc.cfg.Security.CertPath
+			opt.KeyPath = rc.cfg.Security.KeyPath
+		}
+		pdController, err := pdutil.NewPdController(ctx, rc.cfg.TiDB.PdAddr, rc.tls.TLSConfig(), opt)
+		if err != nil {
+			return errors.Trace(err)
+		}
+		log.L().Info("removing some PD schedulers")
```

Comment:
```suggestion
		logTask.Info("removing some PD schedulers")
```

similar for all other `log.L()`.